gitpod-io/gitpod
# [supervisor] poll ready check only if workspace is running

#### What it does

- fix #3577: It is not possible to tone down browser errors, but we don't need to poll ready checks if workspace is not running.

#### How to test

- Start a workspace which takes a while to build and pull images. You should not see failed attempts of ready checks while it is happening, but only when a workspace is running.
